### DISCUSSION TITLE
Tab name doesn't exist?

### DISCUSSION THREAD

**Discussion by MimiNendo** [bumbleshoot/quest-tracker#2] [Q&A] [answered]
I created a blank google sheet to use for the quest tracker, and named it on the top left "Vanguard Quest Tracker". 
Here's what the code looks like:

const AUTO_UPDATE_QUEST_TRACKER = true;
const QUEST_TRACKER_SPREADSHEET_URL = "https://docs.google.com/spreadsheets";    *
*(removed the details of the link on this post for privacy purposes)
const QUEST_TRACKER_SPREADSHEET_TAB_NAME = "Vanguard Quest Tracker";

This is a part of the code I intend to use in "Automate Habitica". I did as the instructions said for all of it, it's just this part about the "tab name" that has me confused and the only error I'm getting so far. 
The error reads:
ERROR: QUEST_TRACKER_SPREADSHEET_TAB_NAME "Vanguard Quest Tracker" doesn't exist.

I appreciate any help I can get and thank you in advance!

**Comment by MimiNendo** [comment, marked as answer]
I got it to work. 
Basically, at the bottom of the google sheet you will see a tab named "Sheet1" since it's an empty new google sheet. I put that in as the tab name and it worked

**Reply by bumbleshoot** [reply]
Yeah, you got it! That is the default `QUEST_TRACKER_SPREADSHEET_TAB_NAME` provided by the script. The [setup instructions](https://github.com/bumbleshoot/quest-tracker#setup-instructions) also say:

> If you've changed the tab name for the sheet you want to print the Quest Tracker to, paste the tab name inside the quotes where it says const QUEST_TRACKER_SPREADSHEET_TAB_NAME = "";.
